mesh/pmesh.hpp

```diff
@@ -305,12 +318,24 @@ class ParMesh : public Mesh
       return &FaceNbrTransformation;
    }
 
-   /// Return the number of shared faces (3D), edges (2D), vertices (1D)
-   int GetNSharedFaces() const;
+   /// Return the shared face index of a given local face
+   int GetSharedFaceIndexOfLocalFace(int face_idx) const;
 
    /// Return the local face index for the given shared face.
+   int GetLocalFaceIndexOfSharedFace(int s) const { return GetSharedFace(s); }
+
+   /// Return the local face index for the given shared face. This performs the
+   /// same function as GetLocalFaceIndexOfSharedFace.
```

Comment:
Yeah, I should switch the implementation. What I was trying to communicate is that `GetLocalFaceIndexOfSharedFace` should be preferred over `GetSharedFace` (and maybe `GetSharedFace` should be deprecated?) because the name is more descriptive.